greenpithumb/io.py

```diff
@@ -0,0 +1,55 @@
+#import RPi.GPIO as GPIO
```

Comment:
We might want to avoid the `io` name because Python already has a built in library with that name:

https://docs.python.org/2/library/io.html

pi_io?